LibrarySeed.sql

```diff
@@ -0,0 +1,62 @@
+
+CREATE TABLE users(
+    id UUID PRIMARY KEY,
+    username TEXT NOT NUll,
+    password CHAR(60) NOT NULL
+);
+
+CREATE TABLE books(
+    isbn VARCHAR(13),
+    asin VARCHAR(10),
+    title TEXT NOT NULL,
+    author TEXT,
+    publicationDate DATE,
+    thumbnail VARCHAR(2000),
+    cover VARCHAR(2000),
```

Comment:
you might have explained this already but what was the difference between thumbnail and cover?

Reply:
Thumbnail is a scaled down version of the cover, or it can be a different image that is disable when a small image is needed